apps/site/tailwind.config.ts

```diff
@@ -1,5 +0,0 @@
-import twConfig from '@node-core/ui-components/tailwind.config';
-
-twConfig.content.push('../../packages/ui-components/**/*.tsx');
```

Comment:
Will Tailwind still detect the UI components?

Reply:
Locally (on Mac) I have a fully working website